integration_tests/__tests__/__snapshots__/failures-test.js.snap

```diff
@@ -62,7 +62,7 @@ Object {
     Expected value to be truthy, instead received
       false
       
-      at Object.throws (__tests__/assertion-count-test.js:14:17)
+      at throws (__tests__/assertion-count-test.js:14:17)
```

Comment:
It seems like you didn't address this. I would prefer to add a test that confirms the behavior still works correctly for class instances/statics as well as making sure that the new behavior here applies only to top level functions in a module rather than all objects. I still would like this to print `Object.throws` if the function is on an object that is not the module object itself, otherwise that would be a bug.